### PR TITLE
Simplifies code that adds user-agent

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Signed-off-by: Adrian Cole <adrian@tetrate.io>

### CODE REVIEW

File: internal/envoy/http.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Tetrate
```
Comment (codefromthecrypt) [inline internal/envoy/http.go:1]:
```suggestion
// Copyright 2021 Tetrate
```

File: internal/envoy/versions.go [hunk 2]
```diff
@@ -22,17 +22,16 @@ import (
...
 // GetEnvoyVersions returns a version map from a remote URL. eg globals.DefaultEnvoyVersionsURL.
 func GetEnvoyVersions(envoyVersionsURL string) (version.EnvoyVersions, error) {
 	result := version.EnvoyVersions{}
 	// #nosec => This is by design, users can call out to wherever they like!
-	resp, err := transport.Get(envoyVersionsURL)
-	if err != nil {
-		return result, err
+	resp, e := httpGet(envoyVersionsURL)
+	if e != nil {
+		return result, e
```
Comment (mathetake) [inline internal/envoy/versions.go:34]: why not use `err` ? 😄

**Review by mathetake — APPROVED** [review]

File: internal/envoy/versions.go (hunk 2 above)
Comment (codefromthecrypt) [reply internal/envoy/versions.go]: switched everything to this form so no more inconsistency!

**Comment by codefromthecrypt** [comment]
Thanks for the nudge @mathetake

**Pull request merged**